### DISCUSSION TITLE
New RavenDB plugin for Fastify (Node.js)

### DISCUSSION THREAD

**Discussion by drakhart** [ravendb/ravendb#15011] [Show and tell]
Hi, folks!

Here at [NearForm](https://github.com/nearform) we thought that RavenDB and [Fastify](https://github.com/fastify/fastify) would be a perfect match of performance oriented and developer friendly tools, so we went ahead and created [fastify-ravendb](https://github.com/nearform/fastify-ravendb): a Fastify plugin to share one or multiple RavenDB document stores across the whole application, and handle sessions automatically on every request.

Under the hood it uses the great [ravendb-nodejs-client](https://github.com/ravendb/ravendb-nodejs-client), so everything that you know and love from it is still available here.

If you’re a Node.js developer please give it a try and let us know your thoughts. Also don’t hesitate to open any issue, feature request or question in [the bug tracker](https://github.com/nearform/fastify-ravendb/issues), or even a pull request if you want to contribute.

We’ll be happy to hear from you!

P.S.: and if you don’t use [Fastify](https://www.fastify.io/) yet don’t miss the chance to discover one of the best Node.js web frameworks out there!

**Comment by georgiosd** [comment]
Love this, nicely done @drakhart !

**Comment by DejanMilicic** [comment]
Great, I just added it to RavenDB Awesome list 

https://github.com/ravendb/awesome-ravendb#nodejs
